test/utils/test_capture.py

```diff
@@ -64,6 +64,7 @@ def logger():
     "use_amp, amp_type",
     [(True, torch.float16), (True, torch.bfloat16), (False, torch.float16)],
 )
+@pytest.mark.parametrize("gradient_clip_norm", [None, 4.0])
```

Comment:
Might be good to have the unit test somehow actually check if the gradients got clipped. Not super needed though.

Reply:
Done.